_events/2020-05-08.md

```diff
@@ -63,10 +63,23 @@ timetable:
 
 images:
   - /images/events/stuttgart/01.jpg
+  - /images/events/stuttgart/02.jpg
+  - /images/events/stuttgart/03.jpg
+  - /images/events/stuttgart/04.jpg
+  - /images/events/stuttgart/05.jpg
 
 redirect_from:
   - /stuttgart
 
 members:
+  - Pierre Neis
+  - Mariana Podesta
   - Christoph Reinicke
+  - Kerstin Biedermann
+  - Juergen Rambo
+  
+  
 ---
+
+### Join a Meetup in between!
+https://www.xing.com/communities/groups/nummer-play14-stuttgart-1116901
```

Comment:
The url as is is not very nice, you can use something like
`[Join us on Xing](https://www.xing.com/communities/groups/nummer-play14-stuttgart-1116901)`